app/javascript/src/components/Player.tsx

```diff
@@ -48,6 +48,11 @@ interface PlayerState {
   focus?: string
   currentItems: string[]
   currentStats: Stat[]
+  previousState: [{
+    focus?: string
+    currentItems: string[]
+    currentStats: Stat[]
+  }]
```

Comment:
didn't like the separate interface definition for the shape of `previousState` objects?

Reply:
no, just didn't remember how we did it the first time